src/Movies/Movies.js

```diff
@@ -2,12 +2,21 @@ import React from 'react'
 import Card from '../Card/Card'
 import './Movies.css'
 
-const Movies = () => {
+const Movies = ({ movies }) => {
+	const moviesList = movies.map(movie => {
+		return (
+			<Card
+				key={movie.id}
+				title={movie.title}
+				poster={movie.poster_path}
+				rating={movie.average_rating}
+			/>
+		)
+	})
+
 	return (
 		<section className='Movies-container'>
-			<Card title='Brave Little Toaster' averageRating='8' poster='https://image.tmdb.org/t/p/original//sA154deR0X51EcR2lm2FfDczryg.jpg'/>
-			<Card title='Movie2' averageRating='10' poster='https://image.tmdb.org/t/p/original//eDnHgozW8vfOaLHzfpHluf1GZCW.jpg'/>
-			<Card title='Akira' averageRating='10000' poster='https://image.tmdb.org/t/p/original//5KlRFKKSbyCiyYpZSS3A6G5bW0K.jpg'/>
+			{moviesList}
```

Comment:
I like declaring a variable (like we did on line 6) and then using it in the JSX. It looks much cleaner and is easier to read!